openfl-tutorials/interactive_api/PyTorch_Huggingface_transformers_SUPERB/envoy/envoy_config_no_gpu.yaml

```diff
@@ -0,0 +1,7 @@
+params:
```

Comment:
Let's just remove this config, since we have examples of `no_gpu` cases, and it's not mentioned even in README.md.